messages/models.py

```diff
@@ -0,0 +1,29 @@
+from django.db import models
```

Comment:
I think that the migrations file should be in the same PR of the model that present them, it's generated from that